lib/diagnostic.js

```diff
@@ -6,6 +6,13 @@
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let minimum = array.reduce(min);`
 const min = function (/* previous, current */) {
```

Comment:
We're using ```min()``` as a callback for ```reduce()``` so it should take at least two arguments, ```previous``` and ```current```.